lib/dependencies/HarmonyEvaluatedImportSpecifierDependency.js

```diff
@@ -103,7 +103,17 @@ HarmonyEvaluatedImportSpecifierDependency.Template = class HarmonyEvaluatedImpor
 		}
 
 		if (typeof value === "boolean") {
-			source.replace(dep.range[0], dep.range[1] - 1, `${value}`);
+			let newValue = `${value}`;
+
+			const originalSource = source.original().source();
+			if (
+				typeof originalSource === "string" &&
+				/\w/.test(originalSource[dep.range[0] - 1] || "")
+			) {
+				newValue = " " + newValue;
```

Comment:
seems safe to simply always add an extra space. Not sure how "in the hot path" this regex test is, but if we dont want to pay that cost, we can simply have an extra whitespace in a few places (which, given a minifier, would be removed anyway)

Reply:
I considered the solution of "always add a extra space before ${value}".

And I'm hesitate in two fix approaches.

Of course I check the cost of `SourceMapSource.source()`. It is O(1) time cost.

> Not sure how "in the hot path" this regex test is

This bug mostly triggered by importing the compressed file in `node_modules`. The code what is the webpack user writed seems not write `return"field" in Module` because their editor will format it to `return "filed" in Module`(Add a space).
(I'm not sure I get the real meaning of "in the hot path"😂)
If you think add a space is a better way for fixing it. I will change my code.